### ISSUE TITLE
Add statsmodels to dependencies

### ISSUE THREAD

**Issue by elainethale** [NREL/jade#45]
With a fresh conda environment, I got this error from jade:

```
(/home/ehale/environments/dsgflex-2-eedr) [ehale@ed6 20210820_TankT_Shifting_test]$ jade auto-config generic_command commands.txt -c config.json
Traceback (most recent call last):
  File "/home/ehale/environments/dsgflex-2-eedr/bin/jade", line 8, in <module>
    sys.exit(cli())
  File "/home/ehale/.local/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/ehale/.local/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/ehale/.local/lib/python3.7/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ehale/.local/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ehale/.local/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/ehale/.local/lib/python3.7/site-packages/jade/cli/auto_config.py", line 52, in auto_config
    setup_logging("auto_config", None, console_level=level)
  File "/home/ehale/.local/lib/python3.7/site-packages/jade/loggers.py", line 82, in setup_logging
    logging_packages = set(Registry().list_loggers())
  File "/home/ehale/.local/lib/python3.7/site-packages/jade/extensions/registry.py", line 71, in __init__
    self._add_extension(extension)
  File "/home/ehale/.local/lib/python3.7/site-packages/jade/extensions/registry.py", line 80, in _add_extension
    cmod = importlib.import_module(extension["job_configuration_module"])
  File "/home/ehale/environments/dsgflex-2-eedr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/ehale/.local/lib/python3.7/site-packages/jade/extensions/demo/autoregression_configuration.py", line 7, in <module>
    from jade.extensions.demo.autoregression_execution import AutoRegressionExecution
  File "/home/ehale/.local/lib/python3.7/site-packages/jade/extensions/demo/autoregression_execution.py", line 9, in <module>
    from statsmodels.tsa.ar_model import AR
ModuleNotFoundError: No module named 'statsmodels'
```

Looks like statsmodels should be promoted to be a basic jade dependency (rather than optional).

**Comment by elainethale** [comment]
Also `cycler`:

  File "/home/ehale/.local/lib/python3.7/site-packages/jade/extensions/demo/autoregression_configuration.py", line 7, in <module>
    from jade.extensions.demo.autoregression_execution import AutoRegressionExecution
  File "/home/ehale/.local/lib/python3.7/site-packages/jade/extensions/demo/autoregression_execution.py", line 10, in <module>
    import matplotlib
  File "/home/ehale/.local/lib/python3.7/site-packages/matplotlib/__init__.py", line 130, in <module>
    from matplotlib.rcsetup import defaultParams, validate_backend, cycler
  File "/home/ehale/.local/lib/python3.7/site-packages/matplotlib/rcsetup.py", line 33, in <module>
    from cycler import Cycler, cycler as ccycler
ModuleNotFoundError: No module named 'cycler'

**Comment by daniel-thom** [comment]
I attempted to fix this issue in v0.4.9. It looks like this issue may have been due to a conda configuration issue. I will close it. If anyone else see this, please re-open it. There is a workaround if you want to get going immediately:
```
rm ~/.jade-registry
```
It will delete the information that is trying to use statsmodels. If you have registered other Jade extensions, you will need to re-register them. If not, just start running commands again.
